init

```diff
@@ -34,6 +36,10 @@ if [[ $? = 4 ]]; then
   etcdctl set /environment/SCRIPTS-REV  $(cd $SCRIPTDIR && git branch|grep '\*'|cut -d' ' -f2 && cd -)
 fi
 
+if [[ "$VERSION" = "v3" ]]; then
```

Comment:
My main issue here is that setup_credentials  will never run for other version (like itv1) unless explicitly added/